bgpd/bgp_debug.c

```diff
@@ -2276,6 +2300,9 @@ DEFUN_NOSH (show_debugging_bgp,
 		vty_out(vty,
 			"  BGP conditional advertisement debugging is on\n");
 
+	if (BGP_DEBUG(nexthop_group, NEXTHOP_GROUP))
+		vty_out(vty, "  BGP nexthop group debugging is on\n");
```

Comment:
Above you print `BGP Nexthop Group debugging is on` (Nexthop capital first), here not, which one is right?

Reply:
let use lower case